charts/scalardb-analytics-postgresql/values.yaml

```diff
@@ -0,0 +1,163 @@
+    scalar.db.contact_points=${env:SCALAR_DB_CONTACT_POINTS:-}
+
+    # Port number for all the contact points. Default port number for each database is used if empty.
+    scalar.db.contact_port=${env:SCALAR_DB_CONTACT_PORT:-}
+
+    # Credential information to access the database. For Cosmos DB, username isn't used. For DynamoDB, AWS_ACCESS_KEY_ID is specified by the username and AWS_ACCESS_SECRET_KEY is specified by the password.
+    scalar.db.username=${env:SCALAR_DB_USERNAME:-}
+    scalar.db.password=${env:SCALAR_DB_PASSWORD:-}
+
+    # Storage implementation. "cassandra" or "cosmos" or "dynamo" or "jdbc" or "grpc" can be set. Default storage is "cassandra".
+    scalar.db.storage=${env:SCALAR_DB_STORAGE:-}
+
+  # -- Secret name that includes sensitive data such as credentials. Each secret key is passed to Pod as environment variables using envFrom.
+  secretName: ""
+
+  image:
+    # -- Docker image repository of ScalarDB Analytics with PostgreSQL.
+    repository: ghcr.io/scalar-labs/scalardb-analytics-postgresql
+    # -- Specify a image pulling policy.
+    pullPolicy: IfNotPresent
+    # -- Override the image tag whose default is the chart appVersion
+    tag: ""
+
+  # -- Optionally specify an array of imagePullSecrets. Secrets must be manually created in the namespace.
+  imagePullSecrets: [name: reg-docker-secrets]
+
+  strategy:
+    rollingUpdate:
+      # -- The number of pods that can be created above the desired amount of pods during an update
+      maxSurge: 25%
+      # -- The number of pods that can be unavailable during the update process
+      maxUnavailable: 25%
+    # -- New pods are added gradually, and old pods are terminated gradually, e.g: Recreate or RollingUpdate
+    type: RollingUpdate
+
+  service:
+    # -- service types in kubernetes
+    type: ClusterIP
+    # -- Service annotations, e.g: prometheus, etc.
+    annotations: {}
+    ports:
+      postgresql:
+        # -- PostgreSQL public port
+        port: 5432
+        # -- PostgreSQL k8s internal port
+        targetPort: 5432
+        # -- PostgreSQL protocol
+        protocol: TCP
+
+  # -- PodSecurityContext holds pod-level security attributes and common container settings.
+  podSecurityContext:
+    # -- To work ScalarDB Analytics with PostgreSQL properly, you must set "201" to "podSecurityContext.fsGroup".
+    fsGroup: 201
+    seccompProfile:
+      type: RuntimeDefault
+
+  # -- Setting security context at the pod applies those settings to all containers in the pod.
+  securityContext:
+    # -- Capabilities (specifically, Linux capabilities), are used for permission management in Linux. Some capabilities are enabled by default.
+    capabilities:
+      drop:
+        - ALL
+    # -- Containers should be run as a non-root user with the minimum required permissions (principle of least privilege).
+    runAsNonRoot: true
+    # -- The PostgreSQL official image use the "postgres (UID=999)" user by default.
+    runAsUser: 999
```

Comment:
The PostgreSQL container image sets non-root user `postgres` with `UID=999`. To run the container with the non-root user properly, we have to set `UID=999` here.
https://github.com/scalar-labs/docker/blob/main/jdk-postgres/8-15/Dockerfile#L14